Dockerfile.rhel8

```diff
@@ -1,4 +1,6 @@
-FROM ubi8:8-released AS build
+ARG OPENSHIFT_CI
```

Comment:
If we're just doing test runs, should we echo the value of this variable to make sure we're on the right track?